src/Views/MediaViewer.vala

```diff
@@ -540,21 +540,37 @@ public class Tuba.Views.MediaViewer : Gtk.Widget, Gtk.Buildable, Adw.Swipeable {
 	protected bool on_keypress (uint keyval, uint keycode, Gdk.ModifierType state) {
```

Comment:
Wait. Why are you doing _any_ of this instead of using shortcuts? That's why it breaks.

Reply:
They wouldn't get triggered IIRC (Escape didn't either when I tried a few hours ago) I'll take a closer look into it at some point :/

Reply:
Well, if things don't work you investigate why they don't work instead of replacing it with a hack that will break other things later...

It probably didn't work because the widget wasn't focused, or something like that

Reply:
I didn't even consider it a hack to be honest 😭 I found out what I was doing wrong, setting up a ShortcutController fixed my issues. I'll open a separate PR, thanks for point it out!

Reply:
I mean this _was_ the way things worked in GTK3. Except even then you needed extra code to e.g. make sure non-en_US keyboard layouts didn't break shortcuts. (which your func doesn't have either)